vm/rust/src/juno_state_reader.rs

```diff
@@ -41,6 +41,7 @@ struct CachedContractClass {
     pub cached_on_height: u64,
 }
 
+// why Lazy?
```

Comment:
As far as I can remember, `SizedCache::with_size` cannot be evaluated in compile time. For `CLASS_CACHE` to be `static`we need it to be lazily instantiated.